src/main/scala/Chess.scala

```diff
@@ -111,21 +112,32 @@ object Chess {
   def apply(req: ClientOut.AnaDrop): ClientIn =
     Monitor.time(_.chessMoveTime) {
       try {
-        Game(
+        val baseGame = Game(
           req.variant.gameLogic,
           req.variant.some,
           Some(req.fen)
-        ).drop(req.role, req.pos).toOption flatMap { case (game, drop) =>
-          game.pgnMoves.lastOption map { san =>
-            makeNode(
-              game,
-              Uci.WithSan(req.variant.gameLogic, Uci(req.variant.gameLogic, drop), san),
-              req.path,
-              req.chapterId,
-              if (game.situation.playable(false)) game.situation.destinations else Map.empty
-            )
-          }
-        } getOrElse ClientIn.StepFailure
+        )
+        val g: Game = req.halfMove
+          .flatMap(m => {
```

Comment:
This is for dealing with the new halfmove concept for amazons. Rather than removing it and trying to deploy, I think we should hold on and deploy this after the multimove stuff.